we.go

```diff
@@ -562,33 +568,24 @@ func marshalLang(rv map[string]interface{}, r *resp) error {
 }
 
 func main() {
-	var numdays = 3
-
 	if len(config.APIKey) == 0 {
 		log.Fatal("No API key specified. Setup instructions are in the README.")
 	}
 	params = append(params, "key="+config.APIKey)
 
-	for _, arg := range os.Args[1:] {
-		if v, err := strconv.Atoi(arg); err == nil && len(arg) == 1 {
-			numdays = v
-		} else {
-			config.City = arg
-		}
-	}
```

Comment:
Hi @schachmat, below is the code:
~~~
    flag.Parse()
    config.City = city

    for _, arg := range flag.Args() {
                if v, err := strconv.Atoi(arg); err == nil && len(arg) == 1 { 
                        days = v 
                } else {
                        config.City = arg 
                }
        }
~~~

When I run it as `wego -days 1 -city Mumbai 2 Paris`, it returns 2 days' data for Paris and not 1 day data for Mumbai. So `flag.Args()`, which returns non-flag arguments, is replacing the values set earlier. I don't see a straightforward way of doing this other than using `if...else`. But before I go that way, I want to be sure that I'm thinking correct.